randovania/data/json_data/prime3.txt

```diff
@@ -902,25 +935,30 @@ Asset id: 17479488334200413695
 > Dock to Port Observation Deck; Heals? False
```

Comment:
For rooms above:
- Xenoresearch Lab
  - Door to Ventilation Shaft should be a Morph Ball Door
  - Rename to "Morph Ball Door to Ventilation Shaft"
- Ventilation Shaft
  - Both doors are Morph Ball Doors
  - Rename to "Morph Ball Door to Xenoresearch Lab" and "Morph Ball Door to Med Lab Alpha" respectively
- Medlab Alpha
  - Door from Ventilation Shaft should be a Morph Ball Door
    - Rename to "Morph Ball Door to Ventilation Shaft"
  - Tunnel Door to Security Access should be a Morph Ball Door
    - Rename to "Morph Ball Door to Security Access"
  - Blue Door to Security Access should be renamed to "Door to Security Access"